src/ajax.js

```diff
@@ -67,7 +67,9 @@ export function fetcherFactory(timeout = 3000, {request, done} = {}) {
   let fetcher = (resource, options) => {
     let to;
     if (timeout != null && options?.signal == null && !config.getConfig('disableAjaxTimeout')) {
-      to = dep.timeout(timeout, resource);
+      const minTimout = 1500;
+      const normalizedTimeout = timeout > minTimout ? timeout : minTimout;
```

Comment:
Probably we can simplify that:
```suggestion
      const normalizedTimeout = Math.max(timeout, minTimout);
```

Reply:
Thanx, that's a good option. 
For me looks if I simplify it this way, then it would add the need to leave a comment to explain why we using Math.max to filter out too little timeout.
I would say original line looks more self explanatory